recipe/meta.yaml

```diff
@@ -151,28 +151,28 @@ outputs:
         - lib/R/lib
     requirements:
       build:
-        - {{ compiler('m2w64_c') }}          # [win]
-        - {{ compiler('m2w64_cxx') }}        # [win]
+        - {{ compiler('m2w64_c') }}    # [win]
+        - {{ compiler('m2w64_cxx') }}  # [win]
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
-        - llvm-openmp            # [osx]
```

Comment:
Why was this dropped?

Reply:
Note that the `cuda` is not dropped - simply had the selector whitespace adjusted.

Reply:
Am not sure about this. For example I see this line in `openblas`:

https://github.com/conda-forge/openblas-feedstock/blob/7dbdf67050d84af131c930b26daa387e7183c7a2/recipe/meta.yaml#L46

This is also the case with `pytorch`:

https://github.com/conda-forge/pytorch-cpu-feedstock/blob/1a3257e9f1aa9cb3ac8d7347c22493024265f821/recipe/meta.yaml#L54-L55

Also a few more feedstocks:

* https://github.com/conda-forge/crest-feedstock/blob/b406e0f50edeb7d67dd0915f63d45159e8ab9c56/recipe/meta.yaml#L30-L31
* https://github.com/conda-forge/fpm-feedstock/blob/ac335692104ce7ca490093144e1da2d1922ce24c/recipe/meta.yaml#L26-L27
* https://github.com/conda-forge/pinocchio-feedstock/blob/415cc6214c2544d3f69a205148b5f3a27df23bcc/recipe/meta.yaml#L43
* https://github.com/conda-forge/pychrono-feedstock/blob/73013dff49b4c89ebf6a891995334decb8b994ce/recipe/meta.yaml#L26-L27

Could you please recheck?

Also let's leave this thread unresolved until we reach consensus. Thanks! 🙏

Reply:
Should add [the conda-forge docs]( https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp ) also suggest OpenMP should be added to `build`

Reply:
Yeah the fact that `run_exports/strong` is used by [`llvm-opemp`]( https://github.com/conda-forge/openmp-feedstock/blob/2e3d1be71c92b2a586ef4c1e3100b1c23587558d/recipe/meta.yaml#L30-L32 ) & [`libgomp`]( https://github.com/conda-forge/ctng-compilers-feedstock/blob/923b25fa711bd71cc7c43299e032bd87579dbce9/recipe/meta.yaml#L440-L442 ) suggests the intended usage is to add both of these to `build` and nowhere else as `run_exports` will add them to `host` & `run` automatically

Could we please make that change here?

Maybe a separate `r-base` issue should be raised to discuss some of the pain points encountered with OpenMP & R to-date

Reply:
Thanks for this clarification! I wasn't previously aware of the strong-weak exporting distinction. 

Perhaps [the problem I was seeing with the R recipes](https://matrix.to/#/!SOyumkgPRWoXfQYIFH:matrix.org/$168693130842PDzPS:gitter.im?via=matrix.org&via=gitter.im&via=cadair.com) was the fact that `llvm-openmp` was included in the `run:` section of the recipe explicitly and that was leading to an overly strict constraint.

I will revert it back here.

Reply:
Ah ok. Yeah adding to `run` seems odd. So it makes sense that it may cause issues

Thanks! 🙏